web/src/components/Content.test.tsx

```diff
@@ -49,4 +49,15 @@ describe("<Content />", () => {
       expect(mockSigninHelper.onSelfRegister).toHaveBeenCalled();
     });
   });
+
+  describe("infoAlert", () => {
+    it("renders info alert with header", () => {
+      const mdString =
+        ":::infoAlert{header='Header Text'}\n" + 'id: "some-id-1"\n' + 'body test"\n' + ':::"\n';
+
+      render(<Content>{mdString}</Content>);
+      expect(screen.getByText("Header Text")).toBeInTheDocument();
+      expect(screen.getByTestId("contentInfoAlert")).toBeInTheDocument();
```

Comment:
I think it might be nice to have an assertion here that expects that someone specific about the Alert component itself is being rendered.

Maybe we could set a default testId in the <Alert> component, and expect that to exist? I see in the component that you can pass one in, but maybe we could have there be a default if you don't pass one, and then look for it here